doc/advanced-configuration/acl.md

```diff
@@ -129,5 +128,4 @@ One can call:
 `acl:match_rule(<<"localhost">>, register, jlib:make_jid(<<"p">>, <<"localhost">>, <<>>)).`<br/>
 
 Which in our case will return deny.
-If the rule is not host specific, one can use `global` instead of `<<"localhost">>`.
-
+If the rule is not host specific, one can use `global` instead of `<<"localhost">>`.
```

Comment:
If the rule is not host specific, you can use `global` instead of `<<"localhost">>`.